test/input.txt

```diff
@@ -148,10 +149,31 @@
   clear
   list
 
+##########################################################
+# test sort person command
+##########################################################
+
+  # should add correctly 
+  add Charlie Dickson pp/333333 e/charlie.d@nus.edu.sg a/333, gamma street t/friends t/school
+  add Esther Potato p/555555 e/esther@not.a.real.potato pa/555, epsilon street t/tubers t/starchy
+  add Betsy Choo pp/222222 pe/benchoo@nus.edu.sg pa/222, beta street t/secretive
+  add Dickson Ee p/444444 pe/dickson@nus.edu.sg a/444, delta street t/friends
+  add Adam Brown p/111111 e/adam@gmail.com a/111, alpha street
+
+  # list data first
+  list
+
+  # sort my data
+  sortname
```

Comment:
Good that you have added test for the new command. Try to work on adding/modifying JUnit test cases.